src/main/java/com/redhat/rhjmc/containerjfr/core/ContainerJfrCore.java

```diff
@@ -52,11 +56,22 @@
 public class ContainerJfrCore {
     private ContainerJfrCore() {}
 
-    public static void initialize() throws CoreException {
+    public static void initialize()
+            throws CoreException, SecurityException, NullPointerException, IOException {
         System.setProperty(
                 "org.openjdk.jmc.common.security.manager",
                 SecurityManager.class.getCanonicalName());
-        LogManager.getLogManager().reset();
+
+        String jmcLoggerLevelProperty = "org.openjdk.jmc.level=" + Level.OFF.getName();
+        String vertxLoggerLevelProperty = "io.vertx.level=" + Level.OFF.getName();
+
+        String config = jmcLoggerLevelProperty + "\n" + vertxLoggerLevelProperty;
```

Comment:
Rather than hardcoding this config here, could the configuration instead be read from a file? The file could be specified by a system property or env var, and if unset, default to these hardcoded values. This way, if these log messages might ever be useful for debugging something, they can be enabled by the deploying end user by simply setting a property/env var and pointing it at a blank configuration file to override this configuration - or pointed at a file that sets the levels to something other than `OFF`.

Reply:
I could change the mapper function to use the old value if it exists, instead of using the new value if it exists (so preference is given to the old value in the case where both exist). That way the user can override these defaults by just setting the same properties in a config file, and pass that in via `-Djava.util.logging.config.file`. (This is the way I had it before, but I went back and forth on it.) And okay, I'll move the defaults to an external file and probably link to it from the README so the user can see which properties they might want to override.

Reply:
I realised that passing in the path to a config file via an env var that sets `-Djava.util.logging.config.file` won't work inside a container. So I think I'll change it so that the env var is the configuration itself, and then just call `updateConfiguration()` function on it.

Reply:
Well, it still could work, but the user would need to mount the config file to the container. That's not really that big of a deal.

Reply:
I think I've got both ways working now. Is there one you'd prefer?

Reply:
I think I prefer the file route, simply because setting potentially multiline and long settings like that is a bit unconventional (although it should work) with environment variables.